docs/docs/recipes/manage-users/on-storage.md

```diff
@@ -11,11 +11,39 @@ TODO: `user.id`
 
 ## Integrate with Logto user data
 
-TODO:
+When preparing to take Logto as an identity solution, you may think about approaches to organizing user information and user-related staff.
 
-- why you don't need a user table in the database
-- how to integrate
-- pros and cons
+Traditionally, developers will create a user table in a SQL database containing the profile and other user-related information, then create a many-to-one relation in other tables, a `user` column pointing to the user table. For other identity providers, take Google as an example. Create a column named `google_id` to achieve "sign in with Google" linking. Now, for Logto, you may want to create a column `logto_id`, but it is not a good way since Logto is more than a social provider, it is a solution that can take charge of all user-related staff.
+
+With the help of Logto, **you don't need a user table**.
+
+Here is a common practice (with Logto):
+
+Forget the user table, store user info in Logto:
+
+1. Create `user_id` column in other tables, save Logto's `user.id`, refer to the previous chapter [Identify User](#identify-user).
```

Comment:
`other tables` sounds opaque. Maybe some thing like `parent tables` or `all your reference tables`?

Reply:
But is not a "reference" between 2 tables, hard to describe, `parent tables` and `reference tables` is not precise, I'll merge for now, cannot figure out a name.